modules/navier_stokes/include/utils/FaceCenteredMapFunctor.h

```diff
@@ -43,6 +43,8 @@ class FaceCenteredMapFunctor : public Moose::FunctorBase<T>, public Map
 
   bool hasBlocks(SubdomainID sub_id) const override;
 
+  ValueType evaluate(const FaceInfo * const fi) const;
```

Comment:
why the change

Reply:
I'd like to be able to directly get the value because the `getVelocity` in the RC object actually uses FI directly too.

Reply:
Added!